Documentation/raspi-Docker-GPIO.md

```diff
@@ -0,0 +1,174 @@
+# Control GPIO pins within rootless Docker container on Raspberry Pi
+
+When it comes to controlling the GPIO pins of a Raspberry Pi, the existing tutorials will get you started in a couple of minutes. But most of the code assumes that it runs on bare metal and has sufficient privileges, e. g. to access the GPIO pins.  
+But what if you want to run this code in a Docker container, or even rootless? This page will explain how to set up all the necessary pieces.
+
+## Prerequisites
+
+In order to follow the instructions below, Docker must be installed on the Raspberry Pi. The official Docker documentation contains an excellent starting point for installing Docker either for the 32-bit (`armhf`, [see here][5]) or 64-bit (`arm64`, [see here][6]) OS.
+
+## Containerizing the app
+
+Let's use the following sample code to blink an LED ([taken from the official documentation][2]) and save it under `BlinkLedSample\Program.cs`:
+```csharp
+using System;
+using System.Device.Gpio;
+using System.Threading;
+
+Console.WriteLine("Blinking LED. Press Ctrl+C to end.");
+int pin = 18;
+using var controller = new GpioController();
+controller.OpenPin(pin, PinMode.Output);
+bool ledOn = true;
+while (true)
+{
+    controller.Write(pin, ((ledOn) ? PinValue.High : PinValue.Low));
+    Thread.Sleep(1000);
+    ledOn = !ledOn;
+}
+```
+
+The project file `BlinkLedSample\BlinkLedSample.csproj` looks like this:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Device.Gpio" Version="3.1.0" />
+  </ItemGroup>
+</Project>
+```
+
+As you can see, it is a .NET 8 console app which references the NuGet package `System.Device.Gpio`.
+
+First of all, let's make sure that this compiles nicely by running `dotnet build`:
```

Comment:
```suggestion
First of all, let's make sure that this compiles nicely by running `dotnet build`:

```

Reply:
Thx for solving the various linter issues!